ja-jp/README.md

```diff
@@ -3,7 +3,7 @@
 nginx を Kubernetes クラスターにデプロイしたいとしましょう。私はおそらくターミナルでつぎのようなコマンドをタイプするでしょう。
 
 ```bash
-kubectl run --image=nginx --replicas=3
+kubectl create deployment --image=nginx --replicas=3
```

Comment:
Do we also need to specify the name of the deployment here, as shown in zh-cn/README.md?

Reply:
Oh, yes! I forgot to add the deployment name...